Tests/IntegrationTestBundle/Resources/config/form.xml

```diff
@@ -11,17 +11,17 @@
 	<services>
 		<service id="integrationTestBundle.form.createTopic"
 				class="Craue\FormFlowBundle\Tests\IntegrationTestBundle\Form\CreateTopicForm">
-			<tag name="form.type" alias="createTopic" />
+			<tag name="form.type" />
 		</service>
 
 		<service id="integrationTestBundle.form.createVehicle"
 				class="Craue\FormFlowBundle\Tests\IntegrationTestBundle\Form\CreateVehicleForm">
-			<tag name="form.type" alias="createVehicle" />
+			<tag name="form.type" />
 		</service>
 
 		<service id="integrationTestBundle.form.issue64"
 				class="Craue\FormFlowBundle\Tests\IntegrationTestBundle\Form\Issue64Form">
-			<tag name="form.type" alias="issue64" />

```

Comment:
It wouldn't. This file is loaded only for Symfony >=2.8, the legacy file (with aliases) will be loaded otherwise. But I guess I could also just add back the alias and >=2.8 will ignore it?